docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -60,14 +72,16 @@ following limitations currently exist for SOA Suite domains:
   currently supports the WebLogic MBean trees only.  Support for JRF MBeans has not
   been added yet.
 
+{{% notice note %}}
+For early access customers, with bundle patch access, it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patches installed. However the customers who do not have access to the bundle patches can obtain the Oracle SOA Suite Docker image from `container-registry.oracle.com` as described in below section.
```

Comment:
However the customers who do not have access to the bundle patches can obtain -> However, if you do not have access to the bundled patches, you can obtain

Reply:
I would mention like this -

However, if you do not have access to the bundle patches, you can obtain the Oracle SOA Suite Docker image without bundle patches from `container-registry.oracle.com' as described in below section.

Reply:
I'm fine with that except the very last part of the sentence should be, "from `container-registry.oracle.com', as described below."